CONTRIBUTING.md

```diff
@@ -85,9 +85,9 @@ Next navigate to _Java_ -> _Code Style_ -> _Organize Imports_. Click _Import_ an
 
 #### IDEA Setup
 
-Open the _Preferences_ window, navigate to _Plugins_ and install the [Eclipse Code Formatter Plugin](https://plugins.jetbrains.com/plugin/6546-eclipse-code-formatter).
+Open the _Settings_ window, navigate to _Plugins_ and install the [Eclipse Code Formatter Plugin](https://plugins.jetbrains.com/plugin/6546-eclipse-code-formatter) from the Marketplace.
```

Comment:
I have `_Preferences_` on my macOs IntelliJ :-)

Maybe we should have the two options?

Reply:
@machi1990 which edition of IDEA do you use? I use the community one.

Reply:
I add both